litmus-portal/frontend/src/pages/WorkflowDetails/index.tsx

```diff
@@ -143,13 +145,21 @@ const WorkflowDetails: React.FC = () => {
   // Setting NodeId of first Node in redux for selection of first node in Argo graph by default
   useEffect(() => {
     if (workflow && pod_name === '') {
-      const firstNodeId = JSON.parse(workflow.execution_data as string).nodes[
-        Object.keys(JSON.parse(workflow.execution_data as string).nodes)[0]
-      ].name;
-      nodeSelection.selectNode({
-        ...JSON.parse(workflow.execution_data as string).nodes[firstNodeId],
-        pod_name: firstNodeId,
-      });
+      if (
+        (JSON.parse(
+          workflow.execution_data as string
+        ) as ExecutionData).phase.toLowerCase() !== 'failed'
```

Comment:
can we use this `Failed` from a constants file, changing it later would be easier if the API data changes, also lower case transformation might not be needed in that case.  @Jonsy13, we are following this approach for analytics, it will help in keeping things consistent and easier to change, also if same constants are used they can be tracked.

Reply:
Sure, addressed